controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -292,6 +292,9 @@ func (r *StorageConsumerReconciler) reconcileCephBlockPool() error {
 				},
 				Parameters: map[string]string{
 					"target_size_ratio": ".49",
+					"pg_autoscale_mode": "off",
+					"pg_num":            "128",
+					"pgp_num":           "128",
```

Comment:
This is not the behavior we agreed to.  You should calculate the total num of PG available based on the device count from that get a value for a single unit that should be used here.

Please see the above comment about cephfs for a more detailed exaplantion.

Reply:
Done